src/services/window-service.js

```diff
@@ -34,13 +26,20 @@ export default class WindowService {
    */
   static createWindow(urls) {
     return new Promise((resolve, reject) => {
-      chrome.windows.create({ url: urls, focused: true }, (win) => {
-        if (chrome.runtime.lastError) {
-          reject(chrome.runtime.lastError);
-        } else {
-          resolve(win);
-        }
-      });
+      const payload = {
+        url: urls,
+        focused: true,
+      };
+
+      // Firefox does not support the focused property
+      if (InstallTrigger) {
+        delete payload.focused;
+      }
```

Comment:
I'm inclined to flip how this situation is handled, as there's a potential for data loss if the wake fails (in its current state)

Reply:
Mitigated the potential for data loss somewhat with 79f250c